README.md

```diff
@@ -32,9 +32,29 @@ func main() {
 
 ## Building
 
-You need Docker and then run
-
-    ./script/build
+You need Docker and ``make`` and then run the ``binary`` target. This
+will create binary for all platform in the `bundles` folder. 
+
+```bash
+λ make binary
```

Comment:
nit, but could you change the prompt to a standard `$` for consistency? (here and in the other examples) :heart: